### PR TITLE
feat: add `form-json-schema` package

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to https://github.com/pinussilvestrus/form-json-schema/issues/6

Adds the [form-json-schema ](https://github.com/pinussilvestrus/form-json-schema/issues/6) to the form-js monorepo.

Todo:
* [x] Add `$schema` to our example files
* [x] fixup failing windows CI

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Our CI has some troubles currently, I will retry later.

File: packages/form-json-schema/LICENSE [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2021-present Camunda Services GmbH
```
Comment (vsgoulart) [inline packages/form-json-schema/LICENSE:1]: Can't we make it open source?

**Review by vsgoulart — APPROVED** [review]

File: packages/form-json-schema/LICENSE (hunk 1 above)
Comment (pinussilvestrus) [reply packages/form-json-schema/LICENSE:1]: I used the same license as for the other packages.

**Review by vsgoulart — APPROVED** [review]

**Comment by pinussilvestrus** [comment]
Windows CI currently failing due to https://github.com/nodejs/node/pull/50136

**Comment by pinussilvestrus** [comment]
We probably need to make the schema generation scripts a bit more robust on Windows

image

**Comment by pinussilvestrus** [comment]
Resolved the broken Windows build by downgrading `json-schema-ref-parse` (cf. https://github.com/APIDevTools/json-schema-ref-parser/issues/303)

**Pull request merged**